src/Basho/Riak/Bucket.php

```diff
@@ -60,17 +64,12 @@ public function getName()
     /**
      * Get the R-value for this bucket
      *
-     * Returns the buckets R-value If it is set,
-     * otherwise return the R-value for the client.
+     * Returns the buckets R-value If it is set, otherwise return the R-value for the client.
      *
-     * @return integer
+     * @return int
```

Comment:
That is how it has been for quite some time, not sure I am going to mess with it unless someone finds an issue with the current implementation.